P/pandoc_crossref/build_tarballs.jl

```diff
@@ -46,4 +48,4 @@ dependencies = [
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies, julia_compat="1.6")
```

Comment:
Also here, can't change Julia compat without changing the 6

Reply:
pandoc-crossref appears to be using 4 parts to their version number:

```
crossref_ver = "0.3.11.0"
```

I don't know exactly what that is about, but in Julia BinaryBuilder only supports 3 parts to it right?

So I bumped the version to `v"0.3.12"` here.